src/Label.js

```diff
@@ -43,6 +44,7 @@ const Label = (props) => {
     tag: Tag,
     check,
     inline,
+    disabled,
```

Comment:
For this use case, the disabled class is needed on the label. You'd end up passing it to both the label and the Input. Most of the time, the label does not wrap the Input. When not inline, you pass disabled to the FormGroup and the Input.
The validation library I create does these thing for the developer using contexts.